dts/bindings/clock/st,stm32mp13-pll-clock.yaml

```diff
@@ -44,23 +52,34 @@ properties:
     type: int
     required: true
     description: |
-        Prescaler for PLLx
-        input clock
+        PLLx DIVM1 division factor of the input clock divider
```

Comment:
Actually, the value of the property is the value applied to PLLx DIVM1 plus 1.
So either not mention `DIVM1` here (and ditto for `DIVN`, ... below) or this needs some rewording, something like:
```
        PLLx division factor of the input clock divider (aka DIVM1 + 1)